R/score_model_out.R

```diff
@@ -122,7 +125,7 @@ score_model_out <- function(model_out_tbl, target_observations, metrics = NULL,
 #' Get scoring metrics
 #'
 #' @param forecast A scoringutils `forecast` object (see
-#' [scoringutils::as_forecast()] for details).
+#' [scoringutils::as_forecast_doc_template()] for details).
```

Comment:
I would suggest to use custom link text instead: 

```suggestion
#' [scoringutils' general information on creating a `forecast` object][scoringutils::as_forecast_doc_template()] for details).
```